app/controllers/about_controller.rb

```diff
@@ -5,5 +5,12 @@ def index
     @smart_proxies = SmartProxy.authorized(:view_smart_proxies).includes(:features)
     @compute_resources = ComputeResource.authorized(:view_compute_resources)
     @plugins = Foreman::Plugin.all
+    @providers = ComputeResource.supported_providers.merge(ComputeResource.registered_providers).keys.map do |provider|
```

Comment:
Having a method on `ComputeResource` that would do the merging (such as `all_providers`) would feel better, as there are just few places we should really differentiate those two, and the `all_providers` would be preferred